src/nvim/ex_cmds.c

```diff
@@ -4960,3 +4961,54 @@ void ex_oldfiles(exarg_T *eap)
     }
   }
 }
+
+void ex_trust(exarg_T *eap)
+{
+  char *p = skiptowhite(eap->arg);
+  size_t len = (size_t)(p - eap->arg);
+  char *arg1 = xmemdupz(eap->arg, len);
+
+  bool success = false;
+  bool action_allow = false, action_deny = false, action_remove = false;
+  char *msg = NULL;
+
+  if (len > 2 && STRNCMP(arg1, "++deny", len) == 0) {
```

Comment:
`STRNCMP` does a cast to `char*` (from `char_u*`). Not necessary here.
```suggestion
  if (len > 2 && strncmp(arg1, "++deny", len) == 0) {
```
Same thing few lines below.

Reply:
Using `strcmp` instead, as discussed in the comment above.
Done in 91345bc149d7d5ef65d6c0735cb698ef42c9068c